### PR TITLE
feat: added new array params

### PR DESCRIPTION
**Pull request by christyjacob4**
- Support for new CLI array params
- Updated README and examples to use the new array syntax

### CODE REVIEW

File: templates/cli/README.md.twig [hunk 1]
```diff
@@ -75,6 +75,16 @@ $ {{ language.params.executableName }} users create --email="hello@{{ spec.title
...
 
+To create a Document you can use the following command 
+```sh
+$ {{ language.params.executableName }} database createDocument --collectionId="YOUR COLLECTION ID" --data='A VALID JSON STRING' --read=role:member --read="*" --write=role:guest
+```
+
+### Some Gotchas
+- `data` expects the JSON string to be escaped.
+- If using the wildcard (`*`) read or write permissions , make sure that it is properly escaped using a `\` or by enclosing it in `"*"`.
+- Some arguments like `read` and `write` permissions are expected to be arrays. In the {{ spec.title }} CLI arrays are passed by simply repeating the argument as seen in the `createDocument` example above.
```
Comment (eldadfux) [inline templates/cli/README.md.twig:86]: In the {{ spec.title }} CLI -> `In the {{ spec.title }} CLI, `

File: tests/SDKTest.php [hunk 2]
```diff
@@ -200,7 +200,8 @@ public function testHTTPSuccess()
             throw new \Exception('Failed to fetch spec from Appwrite server');
         }
 
-        $whitelist = ['php', 'cli', 'node', 'ruby', 'python', 'typescript', 'deno', 'dotnet', 'dart', 'flutter'];
+        // $whitelist = ['php', 'cli', 'node', 'ruby', 'python', 'typescript', 'deno', 'dotnet', 'dart', 'flutter'];
+        $whitelist = ['cli'];
```
Comment (eldadfux) [inline tests/SDKTest.php:204]: Need to replace these lines :)

**Review by eldadfux — CHANGES_REQUESTED** [review]

**Review by lohanidamodar — APPROVED** [review]
Apart from the changes @eldadfux requested. Everything else LGTM.

File: tests/SDKTest.php (hunk 2 above)
Comment (christyjacob4) [reply tests/SDKTest.php]: Done :D

File: templates/cli/README.md.twig (hunk 1 above)
Comment (christyjacob4) [reply templates/cli/README.md.twig]: Done :D

**Review by eldadfux — APPROVED** [review]

**Pull request merged**